game_for_class.py

```diff
@@ -9,27 +9,55 @@
 class Character:
     """All the methods the game player can use"""
     def __init__(self,
-                 name: str,
-                 hp: float,
-                 attack: float,
-                 defense: float,
-                 speed: float,
-                 team: int):
-        self.name = name
-        self.hp = hp
-        self.attack = attack
-        self.defense = defense
-        self.speed = speed
-        self.team = team
+
```

Comment:
I think you messed up the formatting here

Reply:
and also removed setting the class attributes. You want to revert this part of the code back